scripts/maintenance/standard-header-guards.pl

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+# USAGE: standard-header-new_guards.pl filename.h >filename.h.new
+#
```

Comment:
Already stale and unnecessary -- the interface is known/defined by this script being a source-maintenance plugin. And the script is not really meant to be used as a stand-alone program.

```suggestion
```

Reply:
from its neighbour sort-includes.pl:

```
# USAGE: sort-includes.pl filename.cc >filename.cc.sorted
```

This said, I have zero attachment to a comment; removing it